api/configuration/ui.ts

```diff
@@ -11,6 +11,8 @@ import {
   FEATURE_SECURE_COOKIE_ENABLED,
   IDAM_CLIENT,
   INDEX_URL,
+  LINKS_MANAGE_CASES_LINK,
```

Comment:
Same here, can remove Manage cases

Reply:
we need to display the MC link in the register page